gremlin-python/src/main/python/gremlin_python/driver/client.py

```diff
@@ -45,15 +46,20 @@ def __init__(self, url, traversal_source, protocol_factory=None,
                  kerberized_service="", headers=None, session=None,
                  enable_user_agent_on_connect=True, **transport_kwargs):
         log.info("Creating Client with url '%s'", url)
+
+        # check via url that we are using http protocol
+        self._use_http = re.search('^http', url)
+
         self._closed = False
         self._url = url
         self._headers = headers
         self._enable_user_agent_on_connect = enable_user_agent_on_connect
         self._traversal_source = traversal_source
-        if "max_content_length" not in transport_kwargs:
+        if not self._use_http and "max_content_length" not in transport_kwargs:
             transport_kwargs["max_content_length"] = 10 * 1024 * 1024
         if message_serializer is None:
             message_serializer = serializer.GraphBinarySerializersV1()
+            # message_serializer = serializer.GraphSONMessageSerializer()
```

Comment:
Intentional?

Reply:
Also something I forgot to remove, thanks!